src/include/units/symbol_text.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <units/bits/external/fixed_string.h>
+
+namespace units {
+
+template<typename StandardCharT, typename ASCIICharT, std::size_t N, std::size_t M>
+struct basic_symbol_text {
+  basic_fixed_string<StandardCharT, N> standard_;
+  basic_fixed_string<ASCIICharT, M> ascii_;
+
+  constexpr basic_symbol_text(StandardCharT s) noexcept: standard_(s), ascii_(s) {}
+  constexpr basic_symbol_text(StandardCharT s, ASCIICharT a) noexcept: standard_(s), ascii_(a) {}
+  constexpr basic_symbol_text(const StandardCharT (&s)[N + 1]) noexcept: standard_(s), ascii_(s) {}
+  constexpr basic_symbol_text(const basic_fixed_string<StandardCharT, N>& s) noexcept: standard_(s), ascii_(s) {}
+  constexpr basic_symbol_text(const StandardCharT (&s)[N + 1], const ASCIICharT (&a)[M + 1]) noexcept: standard_(s), ascii_(a) {}
+  constexpr basic_symbol_text(const basic_fixed_string<StandardCharT, N>& s, const basic_fixed_string<ASCIICharT, M>& a) noexcept: standard_(s), ascii_(a) {}
+
+  [[nodiscard]] constexpr auto& standard() { return standard_; }
+  [[nodiscard]] constexpr const auto& standard() const { return standard_; }
+  [[nodiscard]] constexpr auto& ascii() { return ascii_; }
+  [[nodiscard]] constexpr const auto& ascii() const { return ascii_; }
+
+  [[nodiscard]] constexpr std::size_t size() const noexcept { return standard_.size(); }
+  [[nodiscard]] constexpr const StandardCharT* c_str() const noexcept { return standard_.c_str(); }
```

Comment:
Do we really need this here? I do not think it helps much taking into account that the user should never play with this type by him/her-self. In the library's framework we can just do the correct thing.

Reply:
Alright, so the user is responsible for using symbol.standard().c_str() instead of symbol.c_str(), correct? Do I refactor the code that uses symbol.c_str() to use symbol.standard().c_str()? That is, is it fine as part of the implementation to only offer ASCII-only support with formatting, and not in other cases?

Reply:
Yes, I was explicitly instructed by SG16 (Unicode Study Group) that streams should provide only basic functionality and all the alternatives and formatting should be handled by the `std::format`.

Reply:
Oh, that makes a lot of sense, thanks.